runtime/lua/vim/lsp/inlay_hint.lua

```diff
@@ -34,7 +53,7 @@ function M.on_inlayhint(err, result, ctx, _)
     return
   end
   local bufstate = bufstates[bufnr]
-  if not bufstate or not bufstate.enabled then
+  if not bufstate.enabled then
```

Comment:
Why did you remove the `not bufstate` condition here?

Reply:
Because I changed `bufstates` to a `vim.defaulttable`, which makes every time the indexed `bufstates[bufnr]` does not exist, it will create a new `bufstate` according to the factory method, which makes `bufstates[bufnr] ~= nil` will no longer make sense because it will always be `true`.